src/components/Admin/AIAnalyticsSummary.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Button, Card, Stack, Badge,
+} from '@edx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { AutoFixHigh, Groups } from '@edx/paragon/icons';
+import useAIAnalyticsSummary from '../AIAnalyticsSummary/data/hooks';
+
+const AnalyticsDetailCard = ({ onClose, data }) => (
+  <Card className="mt-3 mb-4">
+    <Card.Section>
+      <Badge variant="light" className="mb-3 font-weight-semibold">
+        Beta
+      </Badge>
+      <Stack gap={1} direction="horizontal">
+        <p className="card-text text-justify small">{data || 'No insights found..'}</p>
+        <Button variant="link" className="mb-4 ml-3" onClick={onClose}>
+          <span className="small font-weight-bold text-gray-800">Dismiss</span>
+        </Button>
+      </Stack>
+      <label className="x-small" htmlFor="poweredBylabel">Powered by OpenAI</label>
+    </Card.Section>
+  </Card>
+);
+
+AnalyticsDetailCard.propTypes = {
+  onClose: PropTypes.func.isRequired,
+  data: PropTypes.string,
+};
+
+const AIAnalyticsSummary = ({ enterpriseId, insights }) => {
+  const [showSummarizeCard, setShowSummarizeCard] = useState(false);
+  const [showTrackProgressCard, setShowTrackProgressCard] = useState(false);
+
+  const analyticsSummary = useAIAnalyticsSummary(enterpriseId, insights);
```

Comment:
It would be more readable to use object deconstruction syntax here.

```
const { data: analyticsSummary, isLoading, error } = useAIAnalyticsSummary(enterpriseId, insights);
```